binr/ragg2/ragg2.c

```diff
@@ -400,6 +405,15 @@ int main(int argc, char **argv) {
 		}
 	}
 
+	// append the provided string
+	if (str) {
+	    int l;
+	    l = strlen (str);
```

Comment:
Define and assign in the same line and use tabs to indent

Reply:
Okay, I'll define and assign in the same line.